plugins/chat/assets/javascripts/discourse/components/chat-upload.gjs

```diff
@@ -54,6 +55,13 @@ export default class ChatUpload extends Component {
     }
   }
 
+  get videoSourceUrl() {
+    const baseUrl = this.args.upload.url;
+    return this.capabilities.isIOS || this.capabilities.isSafari
```

Comment:
You can probably just check for `isSafari` here, no? It should return `true` on iOS at all times.